packages/jest-editor-support/src/types.js

```diff
@@ -0,0 +1,73 @@
+// @flow
```

Comment:
did you check the top level `types/` folder? I think we don't need all of the types here.

Also, check out Flow's exact types:

```
const o = {|
  a: string,
  b string,
|};
```

if you pass `{a: 'a', b: 'b', c: 'c'}` into this it will say that `c` shouldn't be passed. It's nice to lock down objects like that.